spec/rubocop/cop/lint/literal_in_interpolation_spec.rb

```diff
@@ -114,6 +121,72 @@
   it_behaves_like('literal interpolation', '%I[s1 s2]', '[\"s1\", \"s2\"]')
   it_behaves_like('literal interpolation', '%i[s1     s2]', '[\"s1\", \"s2\"]')
   it_behaves_like('literal interpolation', '%i[ s1   s2 ]', '[\"s1\", \"s2\"]')
+  it_behaves_like('literal interpolation', '{"a" => "b"}', '{\"a\"=>\"b\"}')
```

Comment:
I think it might be nice to have the hash-related tests in their own section. The tests for this cop are already quite messy and we should try to improve this.

Reply:
@bbatsov thank you!
I created sctions for each type.
How about this change?
https://github.com/rubocop/rubocop/pull/11475/commits/c9fdcdc8c4a864b8476fe54560721b4fe997fe98